task_1/index.html

```diff
@@ -0,0 +1,34 @@
+<!doctype html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <link rel="stylesheet" href="./css/main.css">
+</head>
+
+<body>
+    <form name="userRegistrationForm" class="registration__form" novalidate>
```

Comment:
have a better differentiating naming convention between classes and ids, either use BEM or stick to use hyphen and underscore interchangeably